jetty-io/src/main/java/org/eclipse/jetty/io/CyclicTimeout.java

```diff
@@ -102,8 +119,9 @@ public boolean schedule(long delay, TimeUnit units)
             if (_timeout.compareAndSet(timeout, new Timeout(newTimeoutAt, wakeup)))
             {
                 if (LOG.isDebugEnabled())
-                    LOG.debug("Installed timeout in {} ms, waking up in {} ms",
+                    LOG.debug("Installed timeout in {} ms, {} wake up in {} ms",
```

Comment:
Use {} on multi-line log

Reply:
I am using `{}`, not sure what you mean here?

Reply:
```java
if (LOG.isDebugEnabled())
{
   LOG.debug(something,
   somethingelse?or:something
   blah,
   blah);
}
```

Reply:
Done.